src/Raygun4php/RaygunClient.php

```diff
@@ -374,6 +377,8 @@ function toJsonRemoveUnicodeSequences($struct) {
      * Does not apply to GET parameters in the request URI.
      * Filters out raw HTTP data in case any filters are defined, since we can't accurately filter it.
      *
+     * @todo Method should be private.
```

Comment:
My suggestion here would be to make a pull request into the master branch, which is now 2.x-dev and accepting of breaking changes, to simply make this a private/protected method.

You could then make a pull request into the `1` branch which would be 1.9.x-dev, becoming the next minor release version, and add `@deprecated 1.9.0 Will become private in 2.x` (or similar).

I don't think using a todo for this is particularly useful to be honest.

If you'd like to update the pull request that would be my suggestion:

* This PR adds new tests
* New PR into 1 to deprecate these (public) methods and add the new equivalents as private
* New PR into master (once the above is merged and merged up to master) to remove the methods entirely

Reply:
I'm not even using this library, so personally, I don't care much.

Reply:
Fair enough. How about we remove these todos then, and I'll get the tests merged in

Reply:
Also, if you could enable contributor access to the pull request then I can make the changes myself. I can also merge this by hand if needed.